### PR TITLE
Track appearance fix

### PR DESCRIPTION
**Pull request by 0xRe1nk0**

### CODE REVIEW

**Comment by sonora** [comment]
Perhaps consider https://github.com/osmandapp/OsmAnd/issues/16509#issuecomment-1491100546 before merging.

**Comment by vshcherb** [comment]
We shouldn't parse individual rules that's too complicated

File: OsmAnd/src/net/osmand/plus/track/TrackDrawInfo.java [hunk 1]
```diff
@@ -143,6 +150,29 @@ public void updateParams(@NonNull OsmandApplication app, @NonNull GpxDataItem gp
 	@Nullable
 	private String getRenderDefaultTrackWidth(@Nullable RenderingRulesStorage renderer) {
 		if (renderer != null) {
+			float defaultWidth = 0.0F;
+			RenderingRule gpxRule = renderer.getRenderingAttributeRule("gpx");
+			Map<Float, String> trackWidthMap = new HashMap<>();
+			if (gpxRule != null && gpxRule.getIfElseChildren().size() > 0) {
+				List<RenderingRule> rules = gpxRule.getIfElseChildren().get(0).getIfChildren().get(0).getIfElseChildren();
```
Comment (vshcherb) [inline OsmAnd/src/net/osmand/plus/track/TrackDrawInfo.java:157]: ???

**Comment by sonora** [comment]
@vshcherb Potential solution without much complexity could be we change our render files to indicate the resulting strokewidth _explicitly_, e.g.
`renderingProperty attr="currentTrackWidth" name="GPX width" description="Width of displayed GPX tracks" type="string" possibleValues="Fine (2),Thin (4),Medium (8),Bold (12)"`
which (as per our already existing code) will cause the button captons on our appearance dialogs to change accordingly, i.e. indicate the resulting strokewidth in parentheses. 

Could even parse these strings to derive the starting value for the "Custom" selection slider if we wanted.

**Pull request closed without merging**